src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/reference/inputs/EceCharacterReferenceDeclaration.vue

```diff
@@ -2,6 +2,9 @@
   <v-row>
     <v-col cols="12" md="12" lg="12" xl="12">
       <v-row no-gutters>
+        <v-col cols="12" v-for="(systemMessage, index) in configStore.$state.systemMessages">
+          <Banner v-if="systemMessage.portalTags && systemMessage.portalTags.includes('REFERENCES')" type="info" :title="systemMessage.message" />
```

Comment:
Does this need to be 

systemMessage.message ? systemMessage.message : ''" ? 

I think this gets around that annoying potentially undefined typescript flag.

Reply:
Oh I know why this didn't flag with the type-check, we need to also import the component into here. 

components: {Banner},